ci-operator/config/openshift-eng/rebasebot/openshift-eng-rebasebot-main.yaml

```diff
@@ -456,6 +456,36 @@ tests:
         requests:
           cpu: 100m
           memory: 200Mi
+- as: kubernetes-nmstate
+  cron: 30 5 * * Mon,Wed,Fri
+  steps:
+    test:
+    - as: kubernetes-nmstate
+      commands: |
+        rebasebot --source https://github.com/nmstate/kubernetes-nmstate:main \
+                  --dest openshift/kubernetes-nmstate:master \
+                  --rebase openshift-networking/kubernetes-nmstate:rebase-bot-master \
+                  --github-app-id 352928 \
+                  --github-app-key /secrets/metal-net-merge-bot/private-key \
+                  --github-cloner-id 352930 \
+                  --github-cloner-key /secrets/metal-net-cloner-bot/private-key \
+                  --git-username metal-net-bot \
+                  --git-email metal-networking@redhat.com \
```

Comment:
Does this alias exist? And do we care?

Reply:
It does not and we don't care. It's simply for the git configuration because git commit needs to have an email. I don't expect anyone ever to send mails there

Reply:
Fair enough!